tools/serverpod_cli/bin/create/create.dart

```diff
@@ -11,24 +11,29 @@ import '../util/print.dart';
 import 'copier.dart';
 import 'port_checker.dart';
 
-const _defaultPorts = <String, int>{
-  'Serverpod API': 8080,
-  'Serverpod insights API': 8081,
-  'Serverpod Relic web server': 8082,
-  'PostgreSQL server': 8090,
-  'Redis server': 8091,
-};
+const _defaultPostgresPort = '8090';
+const _defaultRedisPort = '8091';
 
 Future<void> performCreate(
   String name,
   bool verbose,
   String template,
   bool force,
+  int postgresPort,
```

Comment:
Make optional. Also, with so many parameters, named parameters will be a better choice.